Automattic/jetpack
# A4A: Add "share details" content to the Connection Card

Resolves https://github.com/Automattic/automattic-for-agencies-dev/issues/47

## Proposed changes:

* Replaces the use of the Jetpack TOS component with custom code, so that we can display information on-site when the "share information" link is clicked, instead of linking to an external source.
* Adds `classnames` dependency to the project, for conditional class management.

### Other information:


## Jetpack product discussion



https://github.com/Automattic/automattic-for-agencies-dev/issues/47

## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* Go to `/wp-admin/options-general.php?page=automattic-for-agencies-client`
* Validate the connection card appears the same.
* Click the "share information" link and validate the content is displayed.
* Click the "back to connection" link and validate all is well.

## Screenshots

<img width="1104" alt="Screenshot 2024-04-03 at 4 53 43 PM" src="https://github.com/Automattic/jetpack/assets/10933065/e5bd280b-5d13-4cae-98e2-f589aed1846b">

<img width="1126" alt="Screenshot 2024-04-03 at 5 06 27 PM" src="https://github.com/Automattic/jetpack/assets/10933065/308d1525-f6fb-4638-b8ca-eff9677e8779">

<img width="1113" alt="Screenshot 2024-04-03 at 5 06 32 PM" src="https://github.com/Automattic/jetpack/assets/10933065/35526626-3e40-4d8a-9d0b-83b706d16d1b">
